test/api/cat.spec.js

```diff
@@ -1,79 +1,17 @@
 /* eslint-env mocha */
 /* globals apiClients */
-'use strict'
-
-const expect = require('chai').expect
-const isNode = require('detect-node')
-const fs = require('fs')
 
-const path = require('path')
-const streamEqual = require('stream-equal')
+'use strict'
 
-let testfile
-let testfileBig
+const test = require('interface-ipfs-core')
 
-if (isNode) {
-  testfile = fs.readFileSync(path.join(__dirname, '/../testfile.txt'))
-  testfileBig = fs.createReadStream(path.join(__dirname, '/../15mb.random'), { bufferSize: 128 })
-} else {
-  testfile = require('raw!../testfile.txt')
+const common = {
+  setup: function (cb) {
+    cb(null, apiClients.a)
+  },
+  teardown: function (cb) {
+    cb()
+  }
 }
 
-describe('.cat', () => {
-  it('cat', (done) => {
-    apiClients.a
-      .cat('Qma4hjFTnCasJ8PVp3mZbZK5g2vGDT4LByLJ7m8ciyRFZP', (err, res) => {
-        expect(err).to.not.exist
-
-        let buf = ''
-        res
-          .on('error', (err) => {
-            expect(err).to.not.exist
-          })
-          .on('data', (data) => {
-            buf += data
-          })
-          .on('end', () => {
-            expect(buf).to.be.equal(testfile.toString())
-            done()
-          })
-      })
-  })
-
-  it('cat BIG file', (done) => {
-    if (!isNode) {
-      return done()
-    }
-
-    apiClients.a.cat('Qme79tX2bViL26vNjPsF3DP1R9rMKMvnPYJiKTTKPrXJjq', (err, res) => {
-      expect(err).to.not.exist
-
-      // Do not blow out the memory of nodejs :)
-      streamEqual(res, testfileBig, (err, equal) => {
-        expect(err).to.not.exist
-        expect(equal).to.be.true
-        done()
-      })
-    })
-  })
-
-  describe('promise', () => {
-    it('cat', (done) => {
-      return apiClients.a.cat('Qma4hjFTnCasJ8PVp3mZbZK5g2vGDT4LByLJ7m8ciyRFZP')
-        .then((res) => {
-          let buf = ''
-          res
-            .on('error', (err) => {
-              throw err
-            })
-            .on('data', (data) => {
-              buf += data
-            })
-            .on('end', () => {
-              expect(buf).to.be.equal(testfile.toString())
-              done()
-            })
-        })
-    })
-  })
-})
+test.files(common)
```

Comment:
Wasn't there an issue somewhere about explicitly renaming that to `mfs` across js-ipfs? Having trouble finding that..